### ISSUE TITLE
Questions about nuget package

### ISSUE THREAD

**Issue by Aimeast** [Lexcess/Chartect.IO#9]
Thank you for make this package, it is awesome! But I have 2 questions:
1. Why add a picture `Chartect.IO.png` to project folder? Is it must keep it on project?
2. Why package have many dependencies? Could you please remove them? `System.Collections`, `System.Diagnostics.Debug`, `System.IO`, `System.Resources.ResourceManager`, `System.Runtime`, `System.Runtime.Extensions`?

**Comment by Lexcess** [comment]
You are absolutely right, the image really shouldn't be in the final package, I'll look at doing a release that removes this (at the moment it is being automatically pulled in). 

As regards the Assembly references; these are actually pulled in with nuspec generator with the old .Net profile. I'm looking at updating this package to use the new .Net Standard so this might become a non-issue. 

No specific timeline yet but I'll probably take a look this weekend. Thanks for the feedback!

**Comment by Aimeast** [comment]
Hi @Lexcess 
Any progress here?

**Comment by Aimeast** [comment]
Hi @Lexcess , This is a very useful package, Could you please make a bit pulse for this issue?

**Comment by Lexcess** [comment]
Hello, the backend build service was broken by an update and I hadn't had the time to fix that. I've done so now and started applying the fixes. Hopefully I'll be able to publish a minor update soon.
